Segmentation fault on receiving community request notification

# Bug Report

## Description

```
message_item.nim amISender
SIGSEGV: Illegal storage access. (Attempt to read from nil?)
make: *** [run-macos] Segmentation fault: 11
```

## Steps to reproduce

 1. Create a community
 2. Send invite to a contact
 3. Request an access to that community

#### Expected behavior

No segmentation fault

#### Actual behavior

Segmentation fault

--- a/src/app/modules/main/activity_center/model.nim
+++ b/src/app/modules/main/activity_center/model.nim
@@ -71,7 +71,7 @@ QtObject:
       of NotifRoles.Name: result = newQVariant(activityNotificationItem.name)
       of NotifRoles.Author: result = newQVariant(activityNotificationItem.author)
       of NotifRoles.NotificationType: result = newQVariant(activityNotificationItem.notificationType.int)
-      of NotifRoles.Message: result = if activityNotificationItem.messageItem != nil:
+      of NotifRoles.Message: result = if not activityNotificationItem.messageItem.isNil:
                                         newQVariant(activityNotificationItem.messageItem)
                                       else:
                                         newQVariant()

--- a/src/app/modules/main/activity_center/model.nim
+++ b/src/app/modules/main/activity_center/model.nim
@@ -71,7 +71,10 @@ QtObject:
       of NotifRoles.Name: result = newQVariant(activityNotificationItem.name)
       of NotifRoles.Author: result = newQVariant(activityNotificationItem.author)
       of NotifRoles.NotificationType: result = newQVariant(activityNotificationItem.notificationType.int)
-      of NotifRoles.Message: result = newQVariant(activityNotificationItem.messageItem)
+      of NotifRoles.Message: result = if activityNotificationItem.messageItem != nil:
+                                        newQVariant(activityNotificationItem.messageItem)
+                                      else:
+                                        newQVariant()
       of NotifRoles.Timestamp: result = newQVariant(activityNotificationItem.timestamp)
       of NotifRoles.PreviousTimestamp: result = newQVariant(if index.row > 0:
                                                               self.activityCenterNotifications[index.row - 1].timestamp

--- a/src/app/modules/main/activity_center/module.nim
+++ b/src/app/modules/main/activity_center/module.nim
@@ -143,8 +143,8 @@ method convertToItems*(
     ): seq[notification_item.Item] =
   result = activityCenterNotifications.map(
     proc(notification: ActivityCenterNotificationDto): notification_item.Item =
-      var messageItem =  msg_item_qobj.newMessageItem(nil)
-      var repliedMessageItem =  msg_item_qobj.newMessageItem(nil)
+      var messageItem: MessageItem
+      var repliedMessageItem: MessageItem
       # default section id is `Chat` section
       let sectionId = if notification.communityId.len > 0:
           notification.communityId

